### PR TITLE
[github] Use doxygen 1.9.1 (1.9.3 is buggy for pages), and fix badges.

### PR DESCRIPTION
**Pull request by dlyr**
Hot fix for doc. Hope it'll work.

### CODE REVIEW

**Comment by dlyr** [comment]
Not sure to catch your point.
Do you mean that runnung  downloaded doxygen 1.9.1 will work on CI ?

**Comment by nmellado** [comment]
I mean that the precompiled binaries of doxygen 1.9.1 will not run on CI (I tried, see: https://github.com/poncateam/ponca/actions/runs/3022729202/jobs/4862149190#step:7:5)

``` bash
Run ./doxygen-1.9.1/bin/doxygen ./build/doc/Doxyfile
[2](https://github.com/poncateam/ponca/actions/runs/3022729202/jobs/4862149190#step:7:2)
  ./doxygen-1.9.1/bin/doxygen ./build/doc/Doxyfile
[3](https://github.com/poncateam/ponca/actions/runs/3022729202/jobs/4862149190#step:7:3)
  shell: /usr/bin/bash -e {0}
[4](https://github.com/poncateam/ponca/actions/runs/3022729202/jobs/4862149190#step:7:5)
./doxygen-1.9.1/bin/doxygen: error while loading shared libraries: libclang-9.so.1: cannot open shared object file: No such file or directory
```

**Comment by MathiasPaulin** [comment]
I'm not sure to understand what you mean.
Before PR "Fix doxygen deploy" (#1002), The radium documentation was built with doxygen 1.9.1 in the CI and was well formed (see documentation for the master branch). Doxygen 1.9.3 is known to be buggy on some structured documentation. I remember  (but I can be wrong) that @dlyr opened an issue on doxygen repo to mention that. So, right now, I found a good idea to stay with doxygen 1.9.1 for Radium documentation. Perhaps your problem is due to some configuration of your ponca CI.

**Comment by nmellado** [comment]
Doxygen 1.9.1 works fine if you use ubuntu package. The proposed fix is to used precompiled binaries provided by doxygen, which does not work.

Regarding the changes introduced in #1000.
Rolling back to 1.9.1 will invalidate the changes introduced in #1000, which require 1.9.3 at least.
Sticking to 1.9.3 will break our layout, there is a bug with markdown pages (see issue submitted by @dlyr https://github.com/doxygen/doxygen/issues/9254).

So... I don't know. I guess it is less confusing to rollback the changes introduced by #1000.

**Comment by nmellado** [comment]
Ok, after diving into doxygen repository, it seems that the fix for the markdown pages have been fixed in April. Release 1.9.5 as been published afterwards, so we could try 1.9.5.

**Comment by dlyr** [comment]
Current CI (on master for instance) use doxygen 1.8.17. which is not compatible with doxygen-awesome-css improvement (at least not as done currently on release-candidate).
But I don't see any issue with 1.9.1, which is the version I have posted here #1001 
I have tested this morning, and 1.9.5 do not solve the markdown pages issue.

**Comment by dlyr** [comment]
looks good for me as shown in artifact [here](https://github.com/STORM-IRIT/Radium-Engine/actions/runs/3061506247)

File: .github/workflows/pull-request-ci.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ on:
   pull_request:
     branches:
       - release-candidate
-      - master
```
Comment (nmellado) [inline .github/workflows/pull-request-ci.yml:11]: Is this change part of the PR ?

Comment (dlyr) [reply .github/workflows/pull-request-ci.yml:11]: could have made in another PR, but pull-request-ci do not work well on pr rc -> ci due to [skip ci] and, we need to fix it by hand anyway

Comment (dlyr) [reply .github/workflows/pull-request-ci.yml:11]: I splitted history in a revert commit + the doxy one.

**Review by nmellado — APPROVED** [review]

**Pull request merged**